apps/nova/src/components/Plans/Content.tsx

```diff
@@ -0,0 +1,218 @@
+import * as Tabs from "@radix-ui/react-tabs";
+import React, { useCallback, useMemo, useState } from "react";
+import { Typography } from "@litespace/luna/Typography";
+import { PlanCard } from "@litespace/luna/PlanCard";
+import { AnimatePresence, motion } from "framer-motion";
+import CurvedDashedLine from "@litespace/assets/CurvedDashedLine";
+import cn from "classnames";
+import { useToast } from "@litespace/luna/Toast";
+import { LocalId } from "@litespace/luna/locales";
+import { useFormatMessage } from "@litespace/luna/hooks/intl";
+import { PlansDataProps, Tab } from "@/components/Plans/types";
+
+const PLANS_TITLE_ID_MAP: { [key: number]: LocalId } = {
```

Comment:
Nice! 

I can thing of a lot of things that are amazing about this line. 
- You declared a map from the id and the intl message instead of using (if/else) or (case/switch)
- The name of the variable is declarative and clear. 
- You defined a type for the variable. You also been specific by using the `LocalId` and not regular `string`. 

Let's push it a step further


```suggestion
const PLANS_TITLE_ID_MAP:  Record<number, LocalId> = {
```